packages/@aws-cdk/aws-rds/lib/engine.ts

```diff
@@ -28,4 +29,11 @@ export interface IEngine {
    *   (which means the major version of the engine is also not known)
    */
   readonly parameterGroupFamily?: string;
+
+  /**
+   * Supported Features by the database engine
+   *
+   * @see https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DBEngineVersion.html
+   */
+  readonly features?: EngineFeatures;
```

Comment:
Why add this? I don't think this is used anywhere?

Reply:
oops, had removed it but didn't push yet - yeah it's not needed :)
good catch.